packages-legacy/angular/package.json

```diff
@@ -40,5 +40,12 @@
   },
   "nx-migrations": {
     "migrations": "@nx/angular/migrations.json"
+  },
+  "lerna": {
+    "command": {
+      "publish": {
+        "directory": "{workspaceRoot}/build/packages/angular-legacy"
```

Comment:
This custom directory config is required for all the legacy packages because {projectRoot} doesn't correlate to its build directory output.